core/app/models/spree/address.rb

```diff
@@ -186,6 +189,17 @@ def name=(value)
 
     private
 
+    def validate_name
+      return if firstname.present?
+
+      name_attribute = if Spree::Config[:show_address_deprecated_attributes]
```

Comment:
This line makes me think that we can pick a better configuration name, showing deprecated attributes may not be directly related to model validation of addresses. I'm thinking to something like `Spree::Config[:use_combined_first_and_last_name_in_address]`, do you think it could work?

Reply:
Yes, makes sense 👍